live/src/agama-installer.changes

```diff
@@ -1,3 +1,12 @@
+-------------------------------------------------------------------
+Wed Feb 12 08:04:11 UTC 2025 - Ladislav Slezák <lslezak@suse.com>
+
+- Start root shell in a free terminal (usually tty2)
+- Fixed restarting the x11-autologin service
+- Do not ask to restore the browser session after restarting it
+- Print only kernel errors or more severe messages on the console,
```

Comment:
Please, add a reference to [bsc#1237056](https://bugzilla.suse.com/show_bug.cgi?id=1237056).